exercises/02.test-structure/01.solution.assertions/README.mdx

```diff
@@ -1,4 +1,6 @@
-# Assertions
+# Implement a Declarative API
```

Comment:
Would it be helpful if we mentioned a declarative API _of X_? 

```
# Implement a Declarative `expect` API
```

Also, shouldn't the exercise titles match between problem/solution? Right now we've got:

- Problem: Imperative vs. Declarative Assertions
- Solution: Implement a Declarative API

Reply:
Okay, I've looked at Kent's workshops and it's also different! My bad, I didn't know that.

Reply:
I always make the title between problem/solution the same.

Reply:
I got lucky with the first exercise I found :D 

- Problem (Loaders): https://github.com/epicweb-dev/full-stack-foundations/blob/55cd8df0ca3cb8bbca045a42be73874762ea6a1d/exercises/03.loading/01.problem.loader/README.mdx
- Solution (Loading Data): https://github.com/epicweb-dev/full-stack-foundations/blob/55cd8df0ca3cb8bbca045a42be73874762ea6a1d/exercises/03.loading/01.solution.loader/README.mdx

I've checked the rest of the Full-stack Fundamentals workshop and you're right, the problem/solution titles are the same!

Reply:
I feel that renaming the exercises might help with SEO as titles now include more searchable words but they now bring less value to the student. I like titles like "Loaders" or "Asset imports". I tried to do the same with "Test blocks" and "Hooks" and "Assertions". Sure, those may be too generic to google for but they are so much more referenceable.